src/arrayOfPeople.js

```diff
@@ -38,7 +38,25 @@
  * @return {number[]}
  */
 function arrayOfPeople(people) {
-  // write code here
+  let arrResult = [];
+  for (let i = 0; i < people.length; i++) {
+    if (arrResult.indexOf(people[i]) !== -1) {
+      continue;
+    }
+    let numberEntries = 0;
+    let positionSimbol = 0;
+    let foundPos = 0;
+    while (people.indexOf(people[i], positionSimbol) !== -1) {
```

Comment:
you are doing loop inside of the loop. 

Try simplifying your solution and solving it in one pass.